src/CMakeLists.txt

```diff
@@ -6,11 +6,16 @@ set(_${COMPONENT_NAME}_sources
     ConstantFlow.cc
     GroupVelocityCompute.cc
     module.cc
+    MovingHarmonicPotential.cc
     ParabolicFlow.cc
+    PlanarMovingHarmonicBarrier.cc
```

Comment:
I'm really sorry... looking at these, I realized I made a mess of these names. Could you please rename one more time as:
* `HarmonicBarrier`
* `PlanarHarmonicBarrier`
* `SphericalHarmonicBarrier`